Blackjack2022/Program.cs

```diff
@@ -171,6 +172,12 @@ static void Options()
 
             if (mForMenu.Key == ConsoleKey.D2)
                 settings.foregroundColor = SelectConsoleColor(settings.foregroundColor);
+
+            if (mForMenu.Key == ConsoleKey.D3)
+                Console.WriteLine("FEATURE NOT ADDED YET");

```

Comment:
username needs to be moved from settings to the user save file when betting is added. this way the username can be the name of the save